OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -116,11 +118,29 @@ def enable(hutil):
     workspaceKey = protected_settings.get("workspaceKey")
     proxy = protected_settings.get("proxy")
     vmResourceId = protected_settings.get("vmResourceId")
+    stopOnMultipleConnections = protected_settings.get("stopOnMultipleConnections")
     if workspaceId is None:
         raise ValueError("Workspace ID cannot be None.")
     if workspaceKey is None:
         raise ValueError("Workspace key cannot be None.")
 
+    if stopOnMultipleConnections is not None and stopOnMultipleConnections is True:
+        output_file = tempfile.NamedTemporaryFile('w')
+        output_file.close()
+
+        list_exit_code = ScriptUtil.run_command(hutil, ScriptUtil.parse_args(WorkspaceCheckCommand), OmsAdminWorkingDirectory, 'Check If Already Onboarded', ExtensionShortName, hutil.get_extension_version(), False, interval = 30, output_file.name)
```

Comment:
can we not use the exit code from omsadmin.sh -l .. if not, we should modify the script to do return an appropriate exit code in this case .. we shouldn't be using strings if we can avoid it